css/properties/break-after.json

```diff
@@ -109,8 +109,24 @@
                 "ie": {
                   "version_added": false
                 },
-                "opera": "mirror",
-                "opera_android": "mirror",
+                "opera": [
+                  {
+                    "version_added": "88"
+                  },
+                  {
+                    "version_added": "11.1",
+                    "version_removed": "12.1"
+                  }
+                ],
+                "opera_android": [
+                  {
+                    "version_added": "mirror"
```

Comment:
This doesn't work, but I've put this here because we need a version of Opera Android based on 102 before we can submit this PR.

Reply:
@queengooborg I don't think this kind of situation will happen often, but any thoughts on how we could support mirroring here too? The problem is that different versions of Opera Android should have different upstream browsers, but we only support a single upstream.

Reply:
I did have a PR that introduced this functionality a while back, but I closed it as it presented many new problems with little benefit (not to mention, the PR that it depended on, which did the same for IE+Edge, was closed as well).  While we could change the upstream mirroring target to Opera Desktop instead, it's much more likely for compatibility to match Chrome Android.

Reply:
I guess I'll just wait for a new Opera Android release, leaving this as draft until then.